### PR TITLE
073124.02/subscribe

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Add Subscribe Switch to Review Page

### CODE REVIEW

**Review by maririos — COMMENTED** [review]
Are there any test you could add for this?

**Comment by chidozieononiwu** [comment]
> Are there any test you could add for this?

It's just a direct API call.

File: src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs [hunk 1]
```diff
@@ -87,16 +87,27 @@ public async Task NotifySubscribersOnNewRevisionAsync(ReviewListItemModel review
...
         /// <param name="reviewId"></param>
+        /// <param name="state"></param> true = subscribe, false = unsubscribe
         /// <returns></returns>
-        public async Task ToggleSubscribedAsync(ClaimsPrincipal user, string reviewId)
+        public async Task ToggleSubscribedAsync(ClaimsPrincipal user, string reviewId, bool? state = null)
         {
             var review = await _reviewRepository.GetReviewAsync(reviewId);
             if (PageModelHelpers.IsUserSubscribed(user, review.Subscribers))
             {
+                if (state == true)
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs:97]: Is UI allowing to click subscribe even if user has already subscribed?

**Review by praveenkuttappan — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs (hunk 1 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs:97]: not UI per say. But the API can technically trigger a subscribe call when user is already subscribed.

**Pull request merged**